src/views/Pipeline.vue

```diff
@@ -318,12 +333,22 @@ export default defineComponent({
       })
     },
     async refreshJobs(event: any) {
+      this.isRetrying = true;
       if(this.segmentSelected === 'pending') {
-        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => { event.target.complete() });
+        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
+          if(event) event.target.complete();
+          this.isRetrying = false;
```

Comment:
We have set `isRetrying` to false, but what if the action throws some error.

Reply:
Sir, we do not return any kind of response from the action so according to me, we will always execute `.then()` statement WDYT.